src/Common/ResetMon.h

```diff
@@ -47,8 +47,11 @@
 #include <SysMsgPlugin.h>
 #include <WString.h>
 
+#if defined(CONFIG_IDF_TARGET_ESP32S3)
+#include "esp32s3/rom/rtc.h"
+#else
 #include "esp32/rom/rtc.h"
-
+#endif

```

Comment:
Please use "rom/crc.h" instead of using preprocessor defines, if possible.

Reply:
Done